module/brick.yaml

```diff
@@ -0,0 +1,47 @@
+name: module
+description: Brick that will generate typical v2 module for you. Enjoy!
+
+version: 0.1.0+1
+
+environment:
+  mason: ">=0.1.0-dev.50 <0.1.0"
+
+vars:
+  name:
+    type: string
+    description: Module name
+    default: module
+    prompt: What is the name of your module?
+  contains_routes:
+    type: boolean
+    description: If true - generates empty routes list
+    default: true
+    prompt: Will your module include routes?
+  level:
+    type: enum
+    description: Controls when your module will be initialized.
+    default: 2
+    prompt: What is your module level?
```

Comment:
should we add some additional information about the module levels? i think it can be useful